DigitalLearningSolutions.Web/Views/Login/Index.cshtml

```diff
@@ -37,7 +37,7 @@
                      type="password"
                      spell-check="false"
                      hint-text=""
-                     autocomplete="current-password"
+                     autocomplete="off"

```

Comment:
Does the username field need to be set to `autocomplete="off"` on line 30 too, if I understand the scope of the ticket correctly?